ITSeed-Official/itseed_backend
# feat: integrate with s3

# Description

Integrate with aws s3 to upload files to remote. (original one is saving in local)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

1. login backend
2. upload file in media UI
3. check the uploaded file URL is s3 URL. 

![image](https://user-images.githubusercontent.com/16770691/138145309-e0499023-c03d-4008-99d5-87fadeee402e.png)
The link is https://itseed-official.s3.ap-northeast-1.amazonaws.com/cropped_logo_192x192_be6754aaaa.png

# Checklist:
